### DISCUSSION TITLE
How to recover deleted plot files

### DISCUSSION THREAD

**Discussion by GeorgePapageorgakis** [Chia-Network/chia-blockchain#6308] [Support]
Is this possible or no since the files are too large and will probably need extended MBR indexing? I tried a couple of tools but they are unrecoverable. Any ideas?

**Comment by BrandtH22** [comment]
@GeorgePapageorgakis , since this ticket was opened a number of years ago without further comments it seems this issue has been resolved. 
We will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions or comments.

Note - especially with the latest plotting software drastically improving plot times it is generally more efficient to replot than to try and recover a deleted plot.

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
